layer1/monitor/events/ethdkg.go

```diff
@@ -70,17 +71,21 @@ func ProcessRegistrationOpened(eth layer1.Client, contracts layer1.AllSmartContr
 		"TaskEnd":   registrationTask.GetEnd(),
 	}).Info("Scheduling NewRegisterTask")
 
-	taskRequestChan <- tasks.NewScheduleTaskRequest(registrationTask)
+	ctx := context.Background()
+	_, err = taskHandler.ScheduleTask(ctx, registrationTask, "")
+	if err != nil {
+		return err
+	}
 
 	// schedule DisputeRegistration
 	logEntry.WithFields(logrus.Fields{
 		"TaskStart": disputeMissingRegistrationTask.GetStart(),
 		"TaskEnd":   disputeMissingRegistrationTask.GetEnd(),
 	}).Info("Scheduling NewDisputeRegistrationTask")
 
-	taskRequestChan <- tasks.NewScheduleTaskRequest(disputeMissingRegistrationTask)
+	_, err = taskHandler.ScheduleTask(ctx, disputeMissingRegistrationTask, "")
```

Comment:
Prefer to write if err check as code added to this later may not observe err check. Change to return nil at end after check.

Reply:
Makes sense, will do.